tty_unix.go

```diff
@@ -20,9 +21,14 @@ func (p *Program) initInput() (err error) {
 		if err != nil {
 			return fmt.Errorf("error entering raw mode: %w", err)
 		}
+
+		// OPTIM: We can use hard tabs to optimize cursor movements if the
+		// terminal doesn't have tab expansion enabled.
+		p.useHardTabs = p.previousTtyInputState.Oflag&unix.TABDLY == unix.TAB0
+		p.useBackspace = p.previousTtyInputState.Lflag&unix.BSDLY == unix.BS0
```

Comment:
this does not work in freebsd, openbsd, dragonfly, not sure about zos

Reply:
Hmm let me take a look. Might need to make an exception for those